### PR TITLE
Improve docs on increasing max file handle limits

### PR DESCRIPTION
**Pull request by wohali**
## Checklist

- [X] Documentation is written and is accurate;
- [X] `make check` passes with no errors
- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: src/maintenance/performance.rst [hunk 1]
```diff
@@ -120,20 +120,44 @@ explanation`_.
...
 
-Finally, most \*nix operating systems impose various resource limits on every
-process. If your system is set up to use the Pluggable Authentication Modules
-(`PAM`_) system, increasing this limit is straightforward. For example,
-creating a file named ``/etc/security/limits.d/100-couchdb.conf`` with the
-following contents will ensure that CouchDB can open enough file descriptors
-to service your increased maximum open databases and Erlang ports::
+Most \*nix operating systems impose various resource limits on every process.
+The method of increasing these limits varies, depending on your init system and
+particular os release. The default value for many OSes is 1024 or 4096. On a
```
Comment (flimzy) [inline src/maintenance/performance.rst]: os - > OS?

File: src/maintenance/performance.rst [hunk 2]
```diff
@@ -120,20 +120,44 @@ explanation`_.
...
+system with many databases or many views, CouchDB can very rapidly hit this
+limit.
+
+If your system is set up to use the Pluggable Authentication Modules (`PAM`_)
+system (as is the cate with nearly all modern Linuxes), increasing this limit
```
Comment (flimzy) [inline src/maintenance/performance.rst]: cate - > case

File: src/maintenance/performance.rst [hunk 3]
```diff
@@ -120,20 +120,44 @@ explanation`_.
...
     couchdb      soft      nofile    4096
 
+You may also have to edit the ``/etc/pam.d/common-session`` and
+``/etc/pam.d/common-session-noninteractive`` files to add the line::
+
+    session required pam_limits.so
+
+if it is not already present.
+
+For systemd-based Linuxes (such as CentOS/RHEL 7, Ubuntu 16.04+, Debian jessie
```
Comment (flimzy) [inline src/maintenance/performance.rst]: Maybe include the Debian version number, not just codename?

**Comment by wohali** [comment]
@flimzy all fixed

**Pull request merged**